plugins/inputs/bond/README.md

```diff
@@ -0,0 +1,81 @@
+# Bond Input Plugin
+
+The Bond Input plugin collects bond interface status, bond's slaves interfaces
+status and failures count of bond's slaves interfaces.
+The plugin collects these metrics from `/proc/net/bonding/*` files.
+
+### Configuration:
+
+```toml
+[[inputs.bond]]
+  ## Sets bonding directory path
+  ## If not specified, then default is:
+  bond_path = "/proc/net/bonding"
+
+  ## By default, telegraf gather stats for all bond interfaces
+  ## Setting interfaces will restrict the stats to the specified
+  ## bond interfaces.
+  bond_interfaces = ["bond0"]
+```
+
+### Measurements & Fields:
+
+- bond
+  - status
+
+- bond_slave
+  - failures
+  - status
+
+### Description:
+
+```
+status
+  Status of bond interface or bonds's slave interface (down = 0, up = 1).
+
+failures
+  Amount of failures for bond's slave interface.
+```
+
+### Tags:
+
+- bond
+  - bond
+
+- bond_slave
+  - bond
+  - interface
+
+### Example output:
+
+Configuration:
+
+```
+[[inputs.bond]]
+  ## Sets bonding directory path
+  ## If not specified, then default is:
+  bond_path = "/proc/net/bonding"
+
+  ## By default, telegraf gather stats for all bond interfaces
+  ## Setting interfaces will restrict the stats to the specified
+  ## bond interfaces.
+  bond_interfaces = ["bond0", "bond1"]
```

Comment:
Comment this line out with a single `#`, so the default will be used.

Reply:
done